demo_fastapi/sales.py

```diff
@@ -57,27 +73,118 @@ def create_order(
         )
 
     # if everything ok, make an order
-    order_id = uuid1().int
+    order_id = random.randrange(10**8)  # 12345  # uuid1().int
     order = Order(
         id=order_id, datestamp=datestamp, buyer=buyer, apples=apples, oranges=oranges
     )
+
+    # save to the database
+    cursor = cnxn.cursor()
+    try:
+        cursor.execute(
+            "INSERT INTO ShoppingList VALUES (?,?,?,?,?)",
+            datestamp,
+            buyer,
+            apples,
+            oranges,
+            order_id,
+        )
+        cursor.commit()
+    except pyodbc.DatabaseError as err:
+        raise HTTPException(
+            status_code=500,
+            detail="Error updating database! Recheck your entries.\n" + str(err),
```

Comment:
Isn't pydantic responsible to make sure that my entries are correct? Why would I get this error?